src/views/News/components/NewsList/index.jsx

```diff
@@ -66,42 +40,75 @@ const NewsList = ({ news, personalUnapprovedNews, handleNewsItemEdit, handleNews
     };
   });
 
-  return news.length > 0 || personalUnapprovedNews.length > 0 ? (
-    <Card>
-      {width < BREAKPOINT_WIDTH ? singleHeader : fullHeader}
-      <Grid>
-        <List className={styles.news_list} disablePadding>
-          {personalUnapprovedNews.length > 0 &&
-            personalUnapprovedNews.map((posting) => (
-              <NewsItem
-                posting={posting}
-                unapproved
-                size={width < BREAKPOINT_WIDTH ? 'single' : 'full'}
-                handleNewsItemEdit={handleNewsItemEdit}
-                handleNewsItemDelete={handleNewsItemDelete}
-                key={posting.SNID}
-              />
-            ))}
+  //https://www.pluralsight.com/guides/re-render-react-component-on-window-resize
+  //Excellent resource for handling rerender on resize -Josh
```

Comment:
We probably don't need to have this comment here. I know it wasn't originally you, but it is unnecessary and I would feel free to remove it.